README.md

```diff
@@ -42,6 +40,39 @@ $ make elrond-clean-sources
 $ make elrond-loaded
 ```
 
+### Installing `kmultiversx`
+
+`kmultiversx` is a Python package providing libraries and CLI tools to interact with the semantics.
+To install `kmultiversx` and its dependencies into a virtual environment, run
+
+```
+# from the project's root directory
+poetry -C kmultiversx install
```

Comment:
The "project's root directory" above is ambiguous, but this did not work:
```
elrond-semantics$ poetry -C kmultiversx install
Installing dependencies from lock file

No dependencies to install or update

Installing the current project: kmultiversx (0.1.0)

elrond-semantics$ poetry -C run foundry --help

The command "foundry" does not exist.
```
This also didn't work:
```
elrond-semantics/kmultiversx$ poetry -C kmultiversx install
Installing dependencies from lock file

No dependencies to install or update

Installing the current project: kmultiversx (0.1.0)

elrond-semantics/kmultiversx$ poetry -C run foundry --help

The command "foundry" does not exist.
```

Reply:
Deleting the poetry cache fixed it.